test/libsolidity/semanticTests/revertStrings/empty_v1_yul.sol

```diff
@@ -0,0 +1,12 @@
+pragma abicoder v1;
```

Comment:
Since going through the IR turns on v2 despite this setting here, shouldn't there be a warning about this?